register_user.sh

```diff
@@ -0,0 +1,44 @@
+#
+# This script inserts a new user into the database given a username and password
+# combination.
+#
+
+if [[ "$#" < 2 ]]
+then
+    cat <<-'END_ERROR' >&2
+	Please provide an email and password to insert into the user database
+    USAGE: ./register_user.sh <user> <pass>
+	END_ERROR
+    exit 1
+fi
+
+user="$1"
+pass="$2"
+
+cd "$( dirname "$0" )" || { echo 'cd error' >&2; exit 1; }
+
+if [ ! -e app/config.ini ]
+then
+    cat <<-'END_ERROR' >&2
+	Couldn't find config file 'app/config.ini'
+	Create this file from 'app/config.ini.default' with the credentials
+	to your database and then run this script again.
+	END_ERROR
```

Comment:
These heredocs are a bit annoying since they contain tabs. Why not a multi-line string, like:
```
echo "
        Couldn't find config file 'app/config.ini'
￼	Create this file from 'app/config.ini.default' with the credentials
￼	to your database and then run this script again.
"
```